examples/application/quickstart3.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: cluster.kurator.dev/v1alpha1
+kind: AttachedCluster
+metadata:
+  name: kurator-member1
+  namespace: default
+spec:
+  kubeconfig:
+    name: kurator-member1
+    key: kurator-member1.config
+---
+apiVersion: cluster.kurator.dev/v1alpha1
+kind: AttachedCluster
+metadata:
+  name: kurator-member2
+  namespace: default
+spec:
+  kubeconfig:
+    name: kurator-member2
+    key: kurator-member2.config
+---
+apiVersion: fleet.kurator.dev/v1alpha1
+kind: Fleet
+metadata:
+  name: quickstart
+  namespace: default
+spec:
+  clusters:
+    - name: kurator-member1
+      kind: AttachedCluster
+    - name: kurator-member2
+      kind: AttachedCluster
+---
+apiVersion: apps.kurator.dev/v1alpha1
+kind: Application
+metadata:
+  name: quickstart3
```

Comment:
change these to better name like `git-repo-demo`

Reply:
ok